src/worker/main.ts

```diff
@@ -1 +1,32 @@
-self.postMessage("hello, from worker");
+import indexedDB from "./indexedDBAccess";
+
+export interface ISaveFileEventPayload {
```

Comment:
Redundant `I` in the name.

```suggestion
export interface SaveFileEventPayload {
```

Reply:
Is this the same interface as in `indexedDBAccess`? Can these two interface definitions be merged?

Reply:
You can, but I don't want them to share the same interface as one of them is for DB access while the other is related to UI event